targets/targets.json

```diff
@@ -2688,6 +2688,7 @@
         "inherits": ["RZ_A1XX"],
         "supported_form_factors": ["ARDUINO"],
         "extra_labels_add": ["RZA1H", "MBRZA1H"],
+        "device_has_add": ["USTICKER"],
```

Comment:
`"USTICKER"` is not a valid value in `"device_has"`. Please remove this change.

Reply:
This PR is for `feature-hal-spec-ticker` branch, I think base on the new HAL spec on this feature branch, this is valid